src/acceptance-test/java/org/zalando/nakadi/webservice/utils/TestStreamingClient.java

```diff
@@ -117,11 +115,8 @@ public TestStreamingClient start() {
 
     public boolean close() {
         if (running) {
-            try {
-                connection.disconnect();
-            } finally {
-                running = false;
-            }
+            running = false;
+            connection.disconnect();
```

Comment:
Ah ok, I see now. I took a look at his branch - yes he actually closes HttpUrlConnection. So everything is ok.